.coveralls.yml

```diff
@@ -1 +0,0 @@
-repo_token: "JFDTIRUVOQ8jCM3zcajrZALlpKXyiXGAX"
```

Comment:
This should be private.
For now, I have moved it inside each CI config. In this way we don't need to install pyyaml just to read this line.

Reply:
If it were 'private' it wouldn't be available to PRs from forks so we wouldn't get coverage on them.  (for some definition of private)